docs/PokemonCreation.md

```diff
@@ -0,0 +1,60 @@
+## Pokemon Creation
+
+The two main files that handle Pokemon creation are [`Equations`](../src/components/Equations.ts) and [`Utilities`](../src/components/Utilities.ts), both of which incorporate ['numberMaker'](../node_modules/numbermakr/lib/NumberMakr.d.ts) for random generation of numbers.
+`Equations` includes functions for choosing random wild Pokemon and for creating the actual Pokemon object. `Utilities` facilitates the creating of a wild Pokemon by preparing parameters then calling `Equations` creation of the Pokemon object.
```

Comment:
This is really vague. Someone reading your description of `Utilities` won't really get any useful information.

As I understood your proposal for the layout, there are two sections you wish to discuss: Equations' (directly making a new Pokemon, such as for unit tests) and Utilities' (creating a new one from a wild Pokemon schema). It would make sense to introduce them in that order.

Good documentation should act like a directed acyclic graph of information, such that if you start at the beginning, you can only reach nodes of information whose prerequisites you've already reached. Make a list of what you'd like to explain, then for each of those things list out the prequisites for them, etc. Depth-first-searching the resultant DAG should give you an approximate order of documentation topics. That way you create docs with a reasonable read-through order that never talk about some information without having explained its prerequisites.